
{'date': 'Sun Nov 24 2019 13:22:25.938 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 49
  def pegamoeda(self):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:22:51.659 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 51
  Jogo.moedas -=1
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:24:05.835 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 54
    pegamoeda 
NameError: name 'pegamoeda' is not defined
'''},
{'date': 'Sun Nov 24 2019 13:28:18.846 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  def __init__(self):
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:28:25.8 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  introd = Cena (img = "https://i.imgur.com/Bcnfg0C.png")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:28:34.257 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:28:38.391 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  def _travesseiro(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:28:43.943 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  def _travesseiro(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:28:53.41 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  def _travesseiro(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:29:00.468 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  colete = Elemento (img = linkcolete,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:29:06.267 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    class Jogo:
  module <module> line 19
    	self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
NameError: name 'FOCO' is not defined
'''},
{'date': 'Sun Nov 24 2019 13:29:19.499 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    class Jogo:
  module <module> line 19
    	self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
AttributeError: 'GUI' object has no attribute 'quartotalita'
'''},
{'date': 'Sun Nov 24 2019 13:31:15.236 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    class Jogo:
  module <module> line 19
    	self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
AttributeError: 'GUI' object has no attribute 'quartotalita'
'''},
{'date': 'Sun Nov 24 2019 13:37:37.355 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  botao.vai=Quarto()
  ^
IndentationError: unexpected indent
'''},